ruled_labels/specs_polkadot.yaml

```
@@ -0,0 +1,250 @@
+---
+name: paritytech/polkadot
+description: Import from paritytech/polkadot
+labels:
+  - name: A0-please_review
+    description: Pull request needs code review.
+    color: "666666"
+  - name: A1-needs_burnin
+    description: Pull request needs to be tested on a live validator node before merge. DevOps is notified via matrix
+    color: c7def8
+  - name: A2-insubstantial
+    description: "Pull request requires no code review (e.g., a sub-repository hash update)."
+    color: C6C7F8
+  - name: A3-in_progress
+    description: Pull request is in progress. No review needed at this stage.
+    color: DBC6F8
+  - name: A4-companion
+    description: A PR that should be considered alongside another (usually more comprehensive and detailed) PR.
```

That is not the definition of a companion PR.
```suggestion
    description: A PR that needs a companion PR to merge in parallel for one of its downstream dependencies.
```
In Substrate we have the `needspolkadotpr` label, but there could be another for Cumulus which is used by Substrate and Polkadot.